templates/navigation-submenu.twig

```diff
@@ -0,0 +1,25 @@
+<nav class='nav-submenu'>
+	<div class='nav-submenu-wrapper'>
+		<ul>
+		{% for key,item in item.children %}
```

Comment:
I think it would make more sense to call this parameter `menu` rather than `item` (so it would become `for key,item in menu.children`), the current notation is a bit confusing imo

Reply:
Completely agree!

Reply:
@mleray I was working on this suggestion (initially agreed by me) but if we change the name to `menu` instead `item` each link inside menu will take `menu.*` values but it seems to be a bit inconsistent that, since it's not a menu, it's an link (or item). What do you think?